lib/CL/pocl_llvm_api.cc

```diff
@@ -394,6 +404,11 @@ int pocl_llvm_build_program(cl_program program,
   PreprocessorOptions &po = pocl_build.getPreprocessorOpts();
 
   std::string kernelh;
+#ifdef HSA_RUNTIME_IS_ROCM
```

Comment:
This should be added to device specific flags instead of pushing ROCM stuff to the generic code.